src/app/shared/components/export-csv/export-csv.component.ts

```diff
@@ -136,6 +137,7 @@ export class ExportCsvComponent implements OnDestroy {
           reportTitle: item.label,
           action: section,
           reportType: item.reportType,
+          order: item.order,
```

Comment:
@diamond-darrell - I think that where item.order is null, we should not send order at all instead  of sending it as null